docs/lightning/rtl.md

```diff
@@ -1,5 +1,83 @@
 # Ride the lightning (RTL)
 
-GUI to manage lightning channels
+A GUI to manage lightning channels
 
-work in progress
+## Using Ride the Lightning (RTL) Wallet
+
+### Logging into RTL
+
+Ride the Lightning is a Lightning wallet and node management tool accessible via a web interface, and is built into myNode.
+RTL is a creation of Suheb/saubyk, and complete information can be found at [Github](https://github.com/ride-the-lightning/RTL).
+The developer can be contacted via Twitter at: @suhub__ or @RTL_App.
+
+You can use RTL from any browser that is able to access your myNode installation.
+
+If you are accessing RTL from inside the network on which your myNode installation resides, you can open a browser window and enter <your.mynode.network.IP:3010> into the address bar.
+
+If you are accessing RTL remotely using the premium features: VPN or Tor, you can open a browser window and enter either <your.home.IP.address:3010> or <your.Tor.hidden.service.address:3010>, depending on which premium feature you are using at the time.
+
+Premium users can get all of their Tor Hidden Service addresses by going to the myNode home page and clicking on the "Tor Services" button in the "Services" area of the page.
+
+Once a Lightning wallet [has been created](https://mynodebtc.com/guide/create_lightning_wallet) on your device, the RTL app will be available via a button on the home page.
+
+<center>
+  <figure>
+    <img src="/images/lightning/RTL/RTL_enable_button_comp.png" alt="RTL enable button" style="width: 400px">
+  </figure>
+</center>
+
+After clicking the button, you will be presented with a login page. Use your myNode password to login. If you have not set your own password, the default is "bolt" and should be changed. After Logging in, the main RTL page will be displayed.
+
+![RTL Main Page](/images/lightning/RTL/RTL_main_page_comp.png "RTL Main Page")
+
+### Funding your BTC Wallet
+
+In order to participate in the Lightning Network, you first need to fund your on-chain Bitcoin wallet. To do this, click on "On-chain" on the Navigation Panel on the left. Then click on the "Generate Address" button.
+
+This will generate a new address and corresponding QR code for you to use to fund your on-chain wallet.
+
+### Opening a Channel
+
+Before you can spend Bitcoin via the Lighting network, you first have to open a Lightning channel to another node.
+And, you must first connect to a chosen lightning node as a "peer" before you can open a channel to it.
+A good place to find potential peers/channels is [here](https://1ml.com/node?order=channelcount&active=true).
+
```

Comment:
I find this para too long/verbose. Since you are listing the steps later below, you can shorten it to something like:

To spend bitcoins on the Lightning network, one must first find peers on the network and open a channel with them. Peers can be trusted friends or complete strangers. 1ML.com or myNode Telegram channel are good place to find peers.

Reply:
Ok. Switched to your version.